ui/src/app/NewInventory/NewInventory.tsx

```diff
@@ -195,7 +196,14 @@ const NewInventory: React.FunctionComponent = () => {
               </StackItem>
               <StackItem>
                 <ActionGroup>
-                  <Button variant="primary" isDisabled={true} onClick={handleSubmit}>Add</Button>
+                  <Button
+                    variant="primary"
+                    isDisabled={true}
```

Comment:
The rest of the buttons have `isDisabled` set to `{isSubmitting}`, why is this one hardcoded to `true`?

Reply:
The andpoints for creating an inventory is not available yet. This one here is  placeholder. The rest of the pages have an actual working Add button

Reply:
Ah gotcha, makes sense 👍